src/libs/actions/IOU.js

```diff
@@ -500,9 +485,10 @@ function createSplitsAndOnyxData(participants, currentUserLogin, amount, comment
  * @param {String} comment
  * @param {String} currency
  * @param {String} locale
+ * @param {String} existingGroupChatReportID
  */
-function splitBill(participants, currentUserLogin, amount, comment, currency, locale) {
-    const {groupData, splits, onyxData} = createSplitsAndOnyxData(participants, currentUserLogin, amount, comment, currency, locale);
+function splitBill(participants, currentUserLogin, amount, comment, currency, locale, existingGroupChatReportID = '') {
+    const {groupData, splits, onyxData} = createSplitsAndOnyxData(participants, currentUserLogin, amount, comment, currency, locale, existingGroupChatReportID);
 
     API.write('SplitBill', {
         reportID: groupData.chatReportID,
```

Comment:
@luacmartins As Monil pointed out, we are not passing the `existingGroupChatReportID` to the API here.

Reply:
requesting a change

Reply:
We actually don't need to pass this param to the API. We have both `SplitBill` and `SplitBillAndOpenReport` commands and that's enough for the API to figure out what it needs to do without the need to pass this param.

Reply:
The front end needs the ID to get the groupChat when it's created from within a chat though

Reply:
That makes sense!